Makefile

```diff
@@ -67,7 +67,7 @@ go-cover: go-tests
 tests: test-go
 
 install-go-fmt-tools:
-	@go install github.com/bufbuild/buf/cmd/buf@v1.28.1
+	@go install github.com/bufbuild/buf/cmd/buf@v1.45.0
```

Comment:
Make sure we do the same in the buf Dockerfile

Reply:
they are the same now, dockerfile is on 45